sdk/identity/identity/test/node/clientCertificateCredential.spec.ts

```diff
@@ -64,7 +64,7 @@ describe("ClientCertificateCredential", function() {
         "Request URL doesn't contain expected clientId"
       );
 
-      const queryParams = qs.parse(authRequest.body);
+      const queryParams: any = qs.parse(authRequest.body);
```

Comment:
Wouldnt using `queryParams.client_assertion as string` in the next line solve the problem without us resorting to using `any`?

Reply:
Yeah, one approach would be to change with the types, though my instinct here was to keep us equivalent to the previous behavior (where `qs.parse` returned any).

Reply:
I think this was one other place where we did a workaround- https://github.com/Azure/azure-sdk-for-js/pull/8782

Reply:
> though my instinct here was to keep us equivalent to the previous behavior (where qs.parse returned any).

That makes sense when we look at the past, but looking into the future, when someone else comes across this piece of code, the first instinct is to worry why we needed to cast something as "any"

Reply:
Fixed